Makefile

```diff
@@ -14,6 +14,10 @@ end-to-end-test:
 	sudo docker build -f ./test/end-to-end/Dockerfile -t weld/end-to-end:latest ./test/end-to-end/
 
 	make -C welder-deployment/ repos
+	git clone ../welder-web
```

Comment:
welder-web has to be in welder-deployment folder according to the docker-compose.yml setup.
And the there's no repos folder.
the git clone ../welder-web is to clone repos in current folder.